apps/explorer/src/components/validator-map/WorldMap.tsx

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Mercator } from '@visx/geo';
+import React, { memo } from 'react';
+import * as topojson from 'topojson-client';
+
+import world from '../node-map/topology.json';
+import { type Feature } from '../node-map/types';
+import { MapFeature } from './MapFeature';
+import { ValidatorLocation } from './ValidatorLocation';
+import { type ValidatorMapData } from './types';
+
+// @ts-expect-error: The types of `world` here aren't aligned but they are correct
+const land = topojson.feature(world, world.objects.countries) as unknown as {
+    type: 'FeatureCollection';
+    features: Feature[];
+};
+
+// We hide Antarctica because there will not be validators there:
+const HIDDEN_REGIONS = ['Antarctica'];
```

Comment:
this is interesting!

Reply:
a lot of this is copy/paste 😅

Reply:
did we lose the D/W/M/ALL filters? are we planning on adding them back? don't have a lot of context here, just curious

Reply:
they don't exist on the new design https://www.figma.com/file/kSD0qXCIDZA4KQVNepA5hR/02-Explorer?type=design&node-id=7077-248095&t=SwZDDyCOuicy4adn-0